pages/checkoutPage.js

```diff
@@ -0,0 +1,29 @@
+const { expect } = require('@playwright/test');
+const { getRandomFirstName, getRandomLastName, getRandomPostalCode} = require('../helpers.js');
+
+exports.CheckoutPage = class CheckoutPage {
+  constructor(page) {
+    this.page = page;
+    this.firstNameInput = page.locator('[data-test="firstName"]');
+    this.lastNameInput = page.locator('[data-test="lastName"]');
+    this.postalCodeInput = page.locator('[data-test="postalCode"]')
+    this.continueButton = page.locator('[data-test="continue"]');
+  }
+  
+  async fillForm() {
```

Comment:
I would split each action to separate function, like:
- `fillFirstName()`
- `fillLastName()`
- `fillPostalCode()`
- `verifyFirstName()`
- ...

And then combine them as a whole action in a function like this.
This would give us more flexibility in the future, e.g. for logged in users, to just verify pre-filled data without filling it.